File: src/Tools/Dova.Tools.Java.ClassGenerator/src/main/java/com/github/sejoslaw/dova/ParameterDefinitionModel.java
package com.github.sejoslaw.dova;

import java.util.ArrayList;
import java.util.Collection;

public class ParameterDefinitionModel {
    public String Type;
    public String Name;
    public Collection<TypeParameterModel> TypeParameterModels = new ArrayList<>();
    public String Signature;
}

File: src/Tools/Dova.Tools.DefinitionGenerator/Models/ParameterDefinitionModel.cs
namespace Dova.Tools.DefinitionGenerator.Models;

internal class ParameterDefinitionModel
{
    public string Type { get; set; }
    public string Name { get; set; }
    public IReadOnlyList<TypeParameterModel> TypeParameterModels { get; set; } = new List<TypeParameterModel>();
    public string Signature { get; set; }
}